README.md

```diff
@@ -54,6 +54,40 @@ For the time being, using React is generally preferred for pages which involve i
 
 Much of the CSS present abides to [BEM](http://getbem.com/) conventions but there is also a fair deal that doesn't. We haven't come to a unanimous decision about how to proceed going forward, so please use your own discretion or continue the discussion in the relevant [issue](https://github.com/ppy/osu-web/issues/53).
 
+### Working with separate osu and store subdomains
```

Comment:
I'd just put a single-line comment in `.env.example` saying "enable this and add hosts entries to..."

I don't think we don't need a walkthrough in the readme on how to add a hosts entry :stuck_out_tongue: